swagger.yaml

```diff
@@ -16,6 +20,47 @@ schemes:
 - "https"
 - "http"
 paths:
+  /clock:
+    get:
+      summary: "Get the server's current time"
+      tags:
+        - "clock"
+      produces:
+        - "application/json"
+      responses:
+        "200":
+          description: "successful operation"
+          schema:
+            $ref: "#/definitions/ClockResponse"
+  /auth/login:
+    post:
+      summary: "Log in with a SIWE message and get a session cookie"
+      tags:
+        - "auth"
+      consumes:
+        - "application/json"
+      produces:
+        - "application/json"
+      parameters:
+        - in: "body"
+          name: "message"
+          description: "The plaintext SIWE message"
+          required: true
+          type: "string"
+        - in: "body"
+          name: "signature"
+          description: "An EIP-191 (for EOA) or EIP-1271 (for contract-owned accounts) signature for the SIWE message"
+          required: true
+          type: "string"
+      responses:
+        "200":
+          description: "successful operation"
+          schema:
+            type: "string"
```

Comment:
I *think* the return type is just the session cookie as a string, but correct me if I'm wrong

Reply:
https://github.com/fiatconnect/specification/blob/main/fiatconnect-api.md#32152-responses

Reply:
The body would just be empty. The session cookie will be part of the `Set-Cookie` response header.

Reply:
ah